sbin/root-mount

```diff
@@ -59,7 +60,7 @@ check_firstboot_resize()
 	EXTENT_SIZE=$(lvm vgdisplay sailfish -c | cut -d ":" -f 13)
 	FREE_KB=$(expr $FREE_EXTENTS \* $EXTENT_SIZE)
 
-	if test "$FREE_KB" -le "$LVM_RESERVED_KB"; then
+	if test "$IS_FS_RESIZED" -eq 1; then
```

Comment:
This should be before any of the FREE_EXTEND, EXTEND_SIZE, FREE_KB as those are not needed to be done for checking this condition anymore.